flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/rules/logical/SortFetchZeroRuleTest.scala

```diff
@@ -22,11 +22,12 @@ import org.apache.flink.table.planner.plan.optimize.program.{BatchOptimizeContex
 import org.apache.flink.table.planner.utils.TableTestBase
 
 import org.apache.calcite.plan.hep.HepMatchOrder
+import org.apache.calcite.rel.rules.PruneEmptyRules
 import org.apache.calcite.tools.RuleSets
 import org.junit.jupiter.api.{BeforeEach, Test}
 
-/** Test for [[FlinkLimit0RemoveRule]]. */
-class FlinkLimit0RemoveRuleTest extends TableTestBase {
+/** Test for [[PruneEmptyRules.SORT_FETCH_ZERO_INSTANCE]]. */
+class SortFetchZeroRuleTest extends TableTestBase {
```

Comment:
I don\t think we need to rename it.
The new name says nothing about the intention of the test. 
Instead of renaming I would suggest to change a javadoc something like
Former test for `FlinkLimit0RemoveRule` which now replaced by Calcite's `PruneEmptyRules.SORT_FETCH_ZERO_INSTANCE`

Reply:
fixed @snuyanzin